docs/reference/migration/migrate_5_4.asciidoc

```diff
@@ -0,0 +1,8 @@
+[[breaking-changes-5.4]]
+== Breaking changes in 5.4
+
+[[breaking_54_delete_by_query_changes]]
+[float]
+=== Delete By Query API changes
```

Comment:
So I think this should be "REST API changes" and then DBQ could be a section within that.